lib/Parse/Parser.cpp

```diff
@@ -1016,7 +1017,7 @@ Parser::parseList(tok RightK, SourceLoc LeftLoc, SourceLoc &RightLoc,
     }
     // If we're in a comma-separated list, the next token is at the
     // beginning of a new line and can never start an element, break.
-    if (Tok.isAtStartOfLine() &&
+    if (Tok.isAtStartOfLine() && 
```

Comment:
nit: whitespace.

Reply:
Fixed in https://github.com/apple/swift/pull/22714/commits/faecb8d5f15a3421f661bd606158d0394dbb19be .